src/traces/parcoords/attributes.js

```diff
@@ -133,5 +133,26 @@ module.exports = {
             autoColorDflt: false,
             editTypeOverride: 'calc'
         })
-    )
+    ),
+
+    unselected: {
+        line: {
+            color: {
+                valType: 'color',
+                dflt: '',
+                editType: 'plot',
+                description: 'Sets the color of unselected lines.'
+            },
+            opacity: {
+                valType: 'number',
+                min: 0,
+                max: 1,
+                dflt: 'auto',
+                editType: 'plot',
+                description: 'Sets the opacity of unselected lines.'
```

Comment:
Can we describe `'auto'`? Not the full formula, just qualitatively something like "The default 'auto' decreases the opacity smoothly as the number of lines increases."

Reply:
Good call. Addressed in 47bef38.